### ISSUE TITLE
Add correct image links for Pypi

### ISSUE THREAD

**Issue by adbar** [adbar/trafilatura#469]
Some images are replaced by alt text as they are missing on the Pypi Readme. This could be fixed by
- Adding the images to the manifest file and make sure they are found
- Replacing the dynamic links by static ones (to the docs) in the Readme (probably easier)
